tests/plugins/imagebase/features/caption.js

```diff
@@ -435,6 +435,33 @@
 			onFocus: function( widget ) {
 				assert.isNotMatching( /Enter image caption/, widget.editor.getData() );
 			}
+		} ),
+
+		// #1646
```

Comment:
It should be `// (#1646)` - https://docs.ckeditor.com/ckeditor4/latest/guide/dev_tests.html#tagging-tests